aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -221,9 +231,9 @@ public class Program
 
 ### Start and StopAsync
 
-[Start(IHost)](/dotnet/api/microsoft.extensions.hosting.hostingabstractionshostextensions.start) starts the host synchronously.
+[Start](/dotnet/api/microsoft.extensions.hosting.hostingabstractionshostextensions.start) starts the host synchronously.
 
-[StopAsync(IHost, TimeSpan)](/dotnet/api/microsoft.extensions.hosting.hostingabstractionshostextensions.stopasync) attempts to stop the host within the provided timeout.
+[StopAsync](/dotnet/api/microsoft.extensions.hosting.hostingabstractionshostextensions.stopasync) attempts to stop the host within the provided timeout.
```

Comment:
This one is ambiguous now, there are two StopAsyncs with different params.

Reply:
Is it better to use `StopAsync(TimeSpan)` or `HostingAbstractionsHostExtensions.StopAsync`?

Is it better to use `IHost.StopAsync` in the spots where the extension for `IHost` is referenced?